framework/src/auxkernels/ElementSizeAux.C

```diff
@@ -0,0 +1,42 @@
+/****************************************************************/
+/*               DO NOT MODIFY THIS HEADER                      */
+/* MOOSE - Multiphysics Object Oriented Simulation Environment  */
+/*                                                              */
+/*           (c) 2010 Battelle Energy Alliance, LLC             */
+/*                   ALL RIGHTS RESERVED                        */
+/*                                                              */
+/*          Prepared by Battelle Energy Alliance, LLC           */
+/*            Under Contract No. DE-AC07-05ID14517              */
+/*            With the U. S. Department of Energy               */
+/*                                                              */
+/*            See COPYRIGHT for full restrictions               */
+/****************************************************************/
+
+// MOOSE includes
+#include "ElementSizeAux.h"
+#include "MooseMesh.h"
+
+template<>
+InputParameters validParams<ElementSizeAux>()
+{
+  InputParameters params = validParams<AuxKernel>();
+  params.addRequiredParam("method", MooseEnum("min max"), "The size calculation to perform ('min' or 'max').");
+  params.addClassDescription("Compute the element size using Elem::hmin() or Elem::hmax() from libMesh.");
+  return params;
+}
+
+
+ElementSizeAux::ElementSizeAux(const InputParameters & parameters) :
+    AuxKernel(parameters),
+    _method(getParam<MooseEnum>("method"))
+{
+}
+
+
+Real
+ElementSizeAux::computeValue()
+{
+  if (_method == "min")
+    return _current_elem->hmin();
```

Comment:
Oh!  You meant the element _length_!

I came in here specifically because I didn't know what you meant by "Size".  To that end... maybe Size is not the best choice (I honestly didn't know what it would be measuring).